Assets/MixedRealityToolkit/_Core/Definitions/Devices/DeviceInputType.cs

```diff
@@ -10,61 +10,80 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
     /// </summary>
     public enum DeviceInputType
     {
+        //Physical Input Type       -       Recommended Axis Type
         None = 0,
         Gaze,
         Voice,
         /// <summary>
         /// 6 Dof Pointer with position and rotation.
         /// </summary>
-        SpatialPointer,
+        SpatialPointer,                     // SixDof Axis
         /// <summary>
         /// 3 Dof Pointer with only position.
         /// </summary>
-        PointerPosition,
+        PointerPosition,                    // ThreeDofPosition Axis
         /// <summary>
         /// 3 Dof Pointer with only rotation.
         /// </summary>
-        PointerRotation,
-        PointerClick,
-        ButtonPress,
-        ButtonTouch,
-        ButtonNearTouch,
-        Trigger,
-        TriggerTouch,
-        TriggerNearTouch,
-        TriggerPress,
+        PointerRotation,                    // ThreeDofRotation Axis
+        PointerClick,                       // Digital
+        ButtonPress,                        // Digital
+        ButtonTouch,                        // Digital
+        ButtonNearTouch,                    // Digital
+        SecondaryButtonPress,               // Digital
+        SecondaryButtonTouch,               // Digital
+        SecondaryButtonNearTouch,           // Digital
+        Trigger,                            // Single Axis
+        TriggerTouch,                       // Digital
+        TriggerNearTouch,                   // Digital
+        TriggerPress,                       // Digital
         /// <summary>
         /// 6 DoF Grip with position and rotation.
         /// </summary>
-        SpatialGrip,
+        SpatialGrip,                        // SixDof Axis
         /// <summary>
         /// 3 DoF Grip with only position.
         /// </summary>
-        GripPosition,
+        GripPosition,                       // ThreeDofPosition Axis
         /// <summary>
         /// 3 Dof Grip with only rotation.
         /// </summary>
-        GripRotation,
-        GripPress,
-        GripTouch,
-        GripNearTouch,
-        ThumbStick,
-        ThumbStickPress,
-        ThumbStickTouch,
-        ThumbStickNearTouch,
-        Touchpad,
-        TouchpadTouch,
-        TouchpadNearTouch,
-        TouchpadPress,
-        Select,
-        Start,
-        Menu,
+        GripRotation,                       // ThreeDofRotation Axis
+        Grip,                               // Single Axis
+        GripPress,                          // Digital
+        GripTouch,                          // Digital
+        GripNearTouch,                      // Digital
+        ThumbStick,                         // Dual Axis
+        ThumbStickPress,                    // Digital
+        ThumbStickTouch,                    // Digital
+        ThumbStickNearTouch,                // Digital
```

Comment:
These aren't always digital.  Why did you add the comments but not summary tags?

Reply:
The Action is digital, even if the input isn't.
Press is on or off, touch is on or off, near touch is either 1 or 0 (on or off)
Same on all controllers, whether it's a trackpad or a thumbstick

Reply:
It's not always digital

![image](https://user-images.githubusercontent.com/13334553/43092977-aa57dbb4-8e63-11e8-9982-8878f374cccf.png)